.ci/jobs/e2e-tests-new.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- job:
+    description: Job that runs e2e tests-new against Elastic Cloud on Kubernetes running in a dedicated k8s cluster. This Job is managed by JJB.
+    logrotate:
+      daysToKeep: 7
+      numToKeep: 100
+      artifactDaysToKeep: 5
+      artifactNumToKeep: 10
+    name: cloud-on-k8s-e2e-tests-new
+    project-type: pipeline
+    triggers:
+      - github
+    concurrent: true
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/elastic/cloud-on-k8s
+            branches:
+              - e2e-testing
```

Comment:
There is no branch `e2e-testing` in elastic/cloud-on-k8s

Reply:
I should have added this to pr description - I'd be creating that branch to run the tests in parallel and trigger runs independently from master.

Reply:
In current configuration job will be triggered on every change in Github, and fails because there is no `e2e-testing` branch. 
In this situation it probably make sense to remove 
```
triggers:
      - github
```
In this case you can trigger job manually